src/public/js/video-tagging/optional-tags.html

```diff
@@ -113,27 +114,35 @@
             Polymer.dom(this.$.buttonsContainer).appendChild(btn);  
         }
 
-        window.addEventListener("keyup", (e) => {
+        if(videotagging.hotkeysAssigned){
+            window.removeEventListener("keyup", this.assignHotkeys);
+        } 
+
+        window.addEventListener("keyup",  this.assignHotkeys);
+        videotagging.hotkeysAssigned = true;
+    }, 
+    assignHotkeys(e){
+        let self = videotagging.optionalTags;
             if (e.keyCode >= 49 && e.keyCode <= 57 && !e.shiftKey) {
+                // debugger;
                 let index = e.keyCode - 49;
-                if (index < this.tags.length) {
-                    let tag = this.tags[index];
-                    let btn = this.getBtnByTag(tag);
+                if (index < videotagging.optionalTags.tags.length) {
```

Comment:
Why the change to use `optionalTags`.  or was `this` previously referring to `videotagging.optionTags` ?

Reply:
yah, in the context of the event listener callback we wanted the this to refer to optional tags